### ISSUE TITLE
The app crashes while opening a vault

### ISSUE THREAD

**Issue by gracie95** [cyb3rko/pincredible#14]
Here's the message shown after opening a vault. But, opening other vault was normal. Not for this one. 

P.S. You don't regenerate and refill too much or you will face app crash.

```
android.content.res.Resources$NotFoundException: Resource ID #0xffffffff
	at android.content.res.ResourcesImpl.getValueForDensity(ResourcesImpl.java:259)
	at android.content.res.Resources.getDrawableForDensity(Resources.java:920)
	at android.content.res.Resources.getDrawable(Resources.java:860)
	at b0.f$a.a(Unknown Source:0)
	at com.cyb3rko.pincredible.fragments.PinViewerFragment$colorTableView$1.invoke(SourceFile:71)
	at com.cyb3rko.pincredible.fragments.PinViewerFragment$colorTableView$1.invoke(Unknown Source:14)
	at com.cyb3rko.pincredible.utils.UtilsKt.iterate(Unknown Source:26)
	at com.cyb3rko.pincredible.fragments.PinViewerFragment.colorTableView(Unknown Source:28)
	at com.cyb3rko.pincredible.fragments.PinViewerFragment.access$colorTableView(Unknown Source:0)
	at com.cyb3rko.pincredible.fragments.PinViewerFragment$loadDataIntoTable$2.invokeSuspend(Unknown Source:16)
	at com.cyb3rko.pincredible.fragments.PinViewerFragment$loadDataIntoTable$2.invoke(Unknown Source:8)
	at com.cyb3rko.pincredible.fragments.PinViewerFragment$loadDataIntoTable$2.invoke(Unknown Source:4)
	at a1.a.h0(SourceFile:4)
	at a1.a.t0(SourceFile:86)
	at com.cyb3rko.pincredible.fragments.PinViewerFragment.loadDataIntoTable(SourceFile:84)
	at com.cyb3rko.pincredible.fragments.PinViewerFragment.access$loadDataIntoTable(Unknown Source:0)
	at com.cyb3rko.pincredible.fragments.PinViewerFragment$onViewCreated$1.invokeSuspend(Unknown Source:28)
	at x2.a.resumeWith(Unknown Source:8)
	at a1.a.X(SourceFile:168)
	at a1.a.L(SourceFile:134)
	at com.cyb3rko.pincredible.fragments.PinViewerFragment.onViewCreated(Unknown Source:68)
	at androidx.fragment.app.Fragment.performViewCreated(SourceFile:5)
	at androidx.fragment.app.h0.e(SourceFile:250)
	at androidx.fragment.app.h0.j(SourceFile:198)
	at androidx.fragment.app.b0.A(SourceFile:979)
	at androidx.fragment.app.b0.U(Unknown Source:91)
	at androidx.fragment.app.b0.y(SourceFile:74)
	at androidx.fragment.app.b0$f.run(Unknown Source:3)
	at android.os.Handler.handleCallback(Handler.java:883)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:237)
	at android.app.ActivityThread.main(ActivityThread.java:7948)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1075)
	Suppressed: n3.x: [y0{Cancelling}@b38a91f, Dispatchers.Main.immediate]

```

**Comment by cyb3rko** [comment]
Weird, it seems not to find the background color resource.  
Which app version do you use?  
Have you recently upgraded from an earlier version?

**Comment by cyb3rko** [comment]
I think I've found the reason why it crashes.  
Could you check the new version 0.4.1?

**Comment by gracie95** [comment]
> I think I've found the reason why it crashes. Could you check the new version [0.4.1](https://github.com/cyb3rko/pincredible/releases/tag/v0.4.1)?

Thanks. But, new bug I just submitted [here](https://github.com/cyb3rko/pincredible/issues/15#issue-1602416623)

**Comment by cyb3rko** [comment]
Implemented in [Version 0.4.1](https://github.com/cyb3rko/pincredible/releases/tag/v0.4.1)
